pkg/configurator/mock_client.go

```diff
@@ -5,10 +5,12 @@
 package configurator
 
 import (
-	gomock "github.com/golang/mock/gomock"
-	announcements "github.com/openservicemesh/osm/pkg/announcements"
 	reflect "reflect"
 	time "time"
+
+	gomock "github.com/golang/mock/gomock"
+
+	announcements "github.com/openservicemesh/osm/pkg/announcements"
```

Comment:
Looks this this file can be dropped from this PR.

Reply:
looks like the formatter wants it to be 
`reflect "reflect"`
`time "time"`

`gomock "github.com/golang/mock/gomock"`
`announcements "github.com/openservicemesh/osm/pkg/announcements"`